### PR TITLE
Create image with basic domain model in extension class

### PR DESCRIPTION
**Pull request by vanajamukkara**
1. Create image with basic domain model in ImageBuilders.java extension class and push to registry.
2. Modified ItMiiDomain.java to use the basic mii image 
3. delete the operator and domain image in extension class

Jenkins results are clean on cluster-10 - 
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/50/

and on kind cluster - 
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/39/

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java [hunk 1]
```diff
@@ -31,18 +33,34 @@
...
       System.getProperty("java.io.tmpdir") + "/ittestspvroot");
+
+  // NGINX constants
   public static final String NGINX_RELEASE_NAME = "nginx-release" + BUILD_ID;
   public static final String STABLE_REPO_NAME = "stable";
   public static final String NGINX_CHART_NAME = "nginx-ingress";
+
+  // MII image constants
+  public static final String MII_BASIC_WDT_MODEL_FILE = "model-basicdomain-singlecluster-sampleapp-wls.yaml";
+  public static final String MII_BASIC_IMAGE_NAME = REPO_USERNAME.equals(REPO_DUMMY_VALUE)
```
Comment (rjeberhard) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java:58]: The REPO_NAME will either be a value or empty, so please always add it.  We need this for the kind-based tests where there is no need for a REPO_USERNAME, but we do have a REPO_NAME.

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java]: yes, I just noticed that...fixing and testing on kind jenkins job

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java]: done

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java [hunk 2]
```diff
@@ -84,5 +120,64 @@ public void beforeAll(ExtensionContext context) {
   @Override
   public void close() {
     logger.info("Cleanup images after all test suites are run");
+
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java:123]: @vanajamukkara Did you test this ? when does this method gets invoked, at the end of the first class that creates the image or at the end of the last class that executes ?

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java:123]: yes, its getting executed at the very end of the last class. https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/50/consoleFull

Comment (sankarpn) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java:123]: Great

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java [hunk 3]
```diff
@@ -84,5 +120,64 @@ public void beforeAll(ExtensionContext context) {
...
+   * @return true if image is created successfully
+   */
+  private boolean createMiiBasicImage(String imageName, String imageTag) {
+
+    final String image = imageName + ":" + imageTag;
+
+    // build the model file list
+    final List<String> modelList = Collections.singletonList(MODEL_DIR + "/" + MII_BASIC_WDT_MODEL_FILE);
+
+    // build an application archive using what is in resources/apps/APP_NAME
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java:149]: can you log this as a message?

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java:149]: done

**Pull request merged**